assets/sass/components/_alertsAndRestrictions.scss

```
@@ -52,3 +52,40 @@
   margin-bottom: govuk-spacing(5);
   padding: govuk-spacing(5) govuk-spacing(5) 0;
 }
+
+.alert-tag {
+  align-items: center;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 3px solid;
+  color: #000;
+  font-size: 14pt;
+  line-height: 1;
+  margin-right: 0;
+  max-width: unset;
+  padding: 8px 8px;
```

```suggestion
  background-color: #fff;
  border-radius: 5px;
  border: 3px solid;
  color: #000;
  font-size: 14pt;
  margin-right: 0;
  padding: 8px 8px;
```
I think it looks closer to the design with this tweak:
* `align-items` - not relevant as this isn't using flex box
* `line-height: 1` is squashing multi-line text
* `max-width: unset` is giving too much right-hand empty space